src/views/add-team.html

```diff
@@ -8,7 +8,7 @@
   <h1 class="govuk-heading-l">Add a GitHub Team</h1>
 
   <p class="govuk-body">
-    GitHub Teams can be used to manage repository permissions and mentions for groups of members.
+    Teams are groups of organisation members that reflect your company or group's structure with cascading access permissions and mentions.
```

Comment:
Very minor, but the commit message duplicates the branch name. I would rename the commit to make it more semantic `Update Add Team description to be consistent with the style from teams documentation on GitHub` or equivalent